wordpress-mobile/gutenberg-mobile
# Implement default block.

This PR updates the `Gutenberg-web` reference to test the default block integration.

![default_block_demo](https://user-images.githubusercontent.com/9772967/48631680-d6578f80-e99d-11e8-8134-5e46e4df3c49.gif)

**To test:**
- Run the project.
- Remove all the blocks with the `trash` icon.
- Check that after removing the last block, a paragraph block appears.
